### DISCUSSION TITLE
Dates / Frequency of release of videos

### DISCUSSION THREAD

**Discussion by imflash217** [OpenMined/courses#59] [General]
Hi @yemikifouly,

At what frequency will the course videos be released?
Is there any schedule of release of the videos?
I have finished the first 3 videos but I can't access the next videos. It would be very helpful to schedule our time if we know the schedule.

Thanks
Vinay

**Comment by yemikifouly** [comment]
@imflash217 Hello Vinay! 🙂 All the videos have been uploaded and are now available for you to view. If you have trouble moving to the next video or concept, it is most probably due to you not having yet attempted the quiz on the current page. So, scroll down. You should see a quiz. Do it and you should be able to move on.

Do let me know if it doesn't work out. 🙂

**Reply by imflash217** [reply]
Oh. I see. The Quiz portion was hidden. Also, the bottom portion of the video gets hidden with the bottom bar too (just a bit). I believe some small CSS changes can make it more accessible to us.
Thanks a lot @yemikifouly 🙂
